front/src/store/modules/ReristrationUser.store.js

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+
+export default {
+  actions: {
+    AddUser ({ commit }, newUser) {
+      axios({
+        method: 'post',
+        url: 'https://localhost:44333/api/user',
+        data: {
+          userName: newUser.userName,
+          email: newUser.email,
+          userPassword: newUser.userPassword
+        }
+      }).then(response => {
+        if (Number(response.data)) {
+          commit('updateMailboxBusy', !response.data);
+          commit('updateVisibleRegistrationForm', !response.data);
+          commit('updateVisibleRegistrationSuccess', response.data);
+        } else {
+          commit('updateMailboxBusy', !response.data);
+        }
+      });
+    }
+  },
+  mutations: {
+    updateMailboxBusy (state, data) {
+      state.mailboxBusy = data;
+    },
+    updateVisibleRegistrationForm (state, data) {
+      state.isVisibleRegistrationForm = data;
+    },
+    updateVisibleRegistrationSuccess (state, data) {
+      state.isVisibleRegistrationSuccess = data;
+    }
+  },
+  state: {
+    mailboxBusy: false,
+    isVisibleRegistrationSuccess: false,
+    isVisibleRegistrationForm: false
+  },
+  getters: {
+    getMailboxBusy (state) {
+      return state.mailboxBusy;
+    },
+    getRegistrationSuccess (state) {
+      return state.isVisibleRegistrationSuccess;
+    },
+    getRegistrationForm (state) {
+      return state.isVisibleRegistrationForm;
+    }
```

Comment:
don't use getters just to return state.
read docs of vuex. getters - for some ways to process in reactive way data in state